### PR TITLE
Support for GCM priority

### PR DESCRIPTION
**Pull request by Syhids**

### CODE REVIEW

**Comment by cubicdaiya** [comment]
Thank for your PR. Could you modify [SPEC.md](https://github.com/mercari/gaurun/blob/master/SPEC.md#post-push) also?

**Comment by Syhids** [comment]
Done @cubicdaiya :)

**Comment by Syhids** [comment]
Hey @cubicdaiya, any update on this? :)

File: gcm/message.go [hunk 1]
```diff
@@ -34,6 +35,10 @@ func (m *Message) validate() error {
 		return fmt.Errorf("the message's RegistrationIDs field must not be nil")
 	}
 
+	if string(m.Priority) != "" && m.Priority != "normal" && m.Priority != "high" {
```
Comment (catatsuy) [inline gcm/message.go]: Why do you need to cast? `m.Priority` is a string type. `m.Priority` is assigned `""` as a zero value.

**Comment by yutachaos** [comment]
@cubicdaiya
When will this PR be merged?
We take this PR by ourselves and fork it, so if possible we want you to merge.

**Comment by Syhids** [comment]
@yutachaos feel free to fork this PR

**Pull request closed without merging**